pyseq.py

```diff
@@ -33,34 +33,35 @@
    http://github.com/rsgalloway/pyseq
 """
 
-__version__ = "0.4.0"
-
-
 import os
 import re
 import logging
+import warnings
 from glob import glob
 from datetime import datetime
 
+__version__ = "0.4.0"
+
 # default serialization format string
-gFormat = '%4l %h%p%t %R'
+global_format = '%4l %h%p%t %R'
```

Comment:
I was also thinking this might be helpful to be:
    global_format = os.environ.get("PYSEQ_FORMAT", '%4l %h%p%t %R')